artemis/templating.py

```diff
@@ -17,10 +17,14 @@
 
 
 def dedent(text: str) -> str:
+    if(text==None):
```

Comment:
`if not text`.

BTW, please run linter (according to https://github.com/cert-polska/artemis?tab=readme-ov-file#code-formatting)

Reply:
done so.